src/Components/PostForm.jsx

```diff
@@ -0,0 +1,216 @@
+import React, { Component } from 'react'
+import { Form, Button, Icon, Header, Segment, Container, Sidebar } from 'semantic-ui-react';
+import axios from 'axios';
+import ImageUploader from 'react-images-upload'
+import PropTypes from 'prop-types'
+
+const TopSidebar = ({ visible, message, successMessage, closeButton }) => (
+  <>
+    <Sidebar
+      id={(successMessage === true) ? 'message-topsidebar-success' : 'message-topsidebar-error'}
+      as={Segment}
+      animation='overlay'
+      direction='top'
+      visible={visible}>
+
+      <p>{message}</p>
+      {closeButton}
+    </Sidebar>
+
+  </>
+)
```

Comment:
I'm not a great fan of declaring multiple components in the same file 😳

Reply:
Yes, as per our discussion yesterday we refactored this and split the file into two components - one for Image Upload Message and one for Create Image Entry. Feels much better now :)